swagger/v1/api_spec.json

```diff
@@ -422,6 +422,59 @@
         }
       }
     },
+    "/api/v1/participant-declarations/{id}": {
+      "get": {
+        "summary": "Get single participant declaration",
+        "operationId": "participant_declarations",
+        "tags": [
+          "participants declaration"
+        ],
+        "security": [
+          {
+            "bearerAuth": [
+
+            ]
+          }
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "example": "9ed4612b-f8bd-44d9-b296-38ab103fadd2",
+            "description": "The ID of the participant declaration ID",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A single participant declaration",
+            "content": {
+              "application/vnd.api+json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ParticipantDeclarationsRecordedResponse"
```

Comment:
`ParticipantDeclarationsRecordedResponse` does not sound that great in here - we don't record anything in a get endpoint. 

Should we rename it `ParticipantDeclarationsResponse`, now that we reuse it between get and post endpoints?